rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+# RFC002 - Blog & video content publishing model (aka "Procope")
+
+## Proposed By
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X)) , Carrie Crowe ([@Carrie Crowe](https://thegooddocs.slack.com/team/U01QS8YAHHN))
+
+## Proposal State
+
+- [x] Draft (submitted to GitHub on May 26, 2021)
+- [ ] Under discussion 
+- [ ] Final comment and voting ({{date}})
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal Overview
+
+Producing and hosting blog posts and video content, including publication ownership and models.
+
+### Motivation
+
+Part of promoting good documentation is helping those uncertain about (or overwhelmed by) what constitutes good docs and best practices. We seek to create a space where the community can contribute to the body of knowledge, as it relates to the mission of the Good Docs Project.
+
+We also want to foster community contributors, to elevate voices that aren't elevated and to empower individuals to be thought leaders in their careers.
+
+## Proposal Details
+
+Regarding specifics on dates and number of participants, this proposal represents our blue sky ideal. It's intended to be flexible with reality, but for simplicity's sake we won't keep mentioning that as a caveat throughout this document.
+
+### Scope of proposal
+
+This proposal focuses on "community-driven media," namely **blog posts** and **videos** about individual topics, created by individual community members and based on those individual viewpoints. All these media elements will be curated by the editorial collective, steered by the core publishers (and input from the Project Steering Committee).
+
+The audiences for this media are:
+* Developers and other "non-technical writers" looking to empower themselves, either in using our templates or in other applicable writing advice
+* Technical writers in that same position
+There is no expectation that either audience will also be GDP contributors or community members. Material will always shoot for outside of the contributor sphere.
+
+#### Out of scope
+
+This proposal doesn't cover **white papers**, though it does address some nuance between white papers and blog posts for clarity.
+
+This proposal also doesn't cover the potential for other media like **podcasts**, but that could be a future expansion.
+
+This proposal also doesn't cover social media, but is certainly tethered to that element of community engagement.
+
+#### Aside: defining white papers vs blog posts
+
+In this model, **blog posts** are community contributions to general knowledge about technical communication, an informal stance of Good Docs Project template support, or other points of interest for those interested in GDP's mission and broader content.
+
+They're periodical content, to keep engagement up. They're individually credited, and live on a GDP-branded Medium.com publication.
```

Comment:
I would like to see the blog be sourced on our website, where content can be reposted or shared on other platforms, but ultimately the source of the content lives in our site repository.  Medium.com has a pay for access model and I don't like that we would be putting our content behind their paywall.  The content from our site could certainly be republished on medium.com if there is some advantage to that, but would not be the original content.

Reply:
FYI, the unstated (until now) requirements for a mature blog site that lead to my Medium.com direction:
- seemless reader subscription of notifications
- consistent and plesant presentation of articles, including with accessibilty tools
- integrated commenting without the need for readers to have an additional account
- clear separation of periodical content and white paper content—in presentation, location, subscription, comment ability

The bonuses Medium has:
- Reader highlighting, as community engagement

How can we get the core site accoomodate all that?

Reply:
(I'll add those requirement notes to the RFC when we resolve or get close to resolving this, so it's included in the core doc. I figure for now, inline is fine so I'm not double-sourcing truth.)

Reply:
I don't subscribe to Medium anymore, and when I see an article behind a paywall now, I ignore it.  I would have thought the hard stop would be making people pay for the content, and not on sharing or republishing our content on channels that people could discover it easily.

As for the newly stated feature requests for the system, some of them are easy to implement, some might take some investigation, but if the project is dead set on using Medium.com instead of the existing site, then I won't spend any time working on the website to give a great blogging experience.  I will note that part of the choice to go with Hugo was with the plan that the site would eventually have a blog for the project, this would be implemented after the migration was complete (as it didn't exist on the Jekyll site).

Reply:
I'm not dead-set on Medium. It fits the requirements, though I'll also go back and review the paywall elements to see if that's related to a setting that can be disabled for a given publication. I agree that any barriers like that should be hard blockers for this project—the information should be always freely available. (Which aligns with my personal goals of providing help to marginalized communities in tech.)

Let's explore what we can do with Hugo to achieve these. There might also be a third option we discover as we dive into particulars.

Reply:
After looking further into it, let's see what we can do to replicate the positive features of Medium on the GDP site. I'm sure a lot of poor income folks use various hacks to get around the paywall, like the Twitter referral. But let's not contribute to the gatekeeping problem. (Clearly I misremembered about the paywall variability element.)

Reply:
Another thing to add to the Hugo research pile: what analytics do we have available? (This is also a general content strategy question, but as a general question is out of scope for this proposal.)

Reply:
Unless there's still outstanding questions on functionality (I think Bryan has resolved all these), should we change the above to reflect that the blog will live on the TGDP site and not Medium?

Reply:
I should have those referenced corrected now in the proposal, as referring to our own hosted blog and not on Medium.